app/src/app_common.cc

```diff
@@ -164,6 +164,12 @@ const char* kCpuArchitecture = "x86";
 #else
 #error Unknown operating system.
 #endif  // Operating system
+
+#if FIREBASE_GITHUB_ACTION_BUILD
+const char* kBuildSource = "github_action_built";
+#elif
```

Comment:
This is causing a compilation error.  Should just be `#else`

Reply:
Whoops. that's what I get for thinking I can get away with minor changes without compiling.🤦